runtime/compiler/optimizer/InterpreterEmulator.cpp

```diff
@@ -348,7 +348,7 @@ InterpreterEmulator::maintainStackForGetField()
             TR_OpaqueClassBlock *fieldDeclaringClass = _calltarget->_calleeMethod->getDeclaringClassFromFieldOrStatic(comp(), cpIndex);
 
             avoidFolding = TR::TransformUtil::avoidFoldingInstanceField(
-               baseObjectAddress, fieldSymbol, cpIndex, _calltarget->_calleeMethod, comp());
+                           baseObjectAddress, fieldSymbol, fieldOffset, cpIndex, _calltarget->_calleeMethod, comp());
```

Comment:
There's another call to this overload in JITClientCompilationThread.cpp. I think that should have to be updated as well, or it will cause a build error when building with JIT server enabled

Reply:
Thanks! I missed it!